### ISSUE TITLE
Support adding emojis directly inside a '-enclosed string

### ISSUE THREAD

**Issue by voxpelli** [mattbierner/vscode-emojisense#9]
I want this to result in an emoji-suggestion:

```javascript
console.log(':wa');
```

Currently only this does:

```javascript
console.log(' :wa');
```

Removing the need for an initial space would be 👌

I often add Emojis at the beginning of a string, especially in logs, as it makes it easier to find that line among all of the lines of a project. Before I've used the Atom equivalent of this module.

**Comment by mattbierner** [comment]
Yeah this on is explicitly disabled right now because it can cause a lot of false positives, such as triggering emoji intellisense when you type `:` in `{'a': b }`. Two possible solutions that don't require understanding the language:

- Show intellisense in this case if you manually request it (hitting ctrl+space)
- Make restriction on the previous character less restrictive. It could instead match a `:` preceded by any non-alphanumeric character perhaps. Still would be broken for `{'a': b }`

The first seems like the better solution

**Comment by voxpelli** [comment]
First one seems simple, but wouldn't it be possible to know if it's a string or not from each language grammar/parser? An always allow this syntax within string contexts?
